drivers/SmartThings/zwave-switch/src/inovelli-vzw32-sn/init.lua

```diff
@@ -0,0 +1,350 @@
+  notificationValue = notificationValue + (255*1)
+  return notificationValue
+end
+
+local function set_color(driver, device, command)
+  device:emit_event(capabilities.colorControl.hue(command.args.color.hue))
+  device:emit_event(capabilities.colorControl.saturation(command.args.color.saturation))
+  device:emit_event(capabilities.switch.switch("on"))
+  local dev = device:get_parent_device()
+  local config = Configuration:Set({
+    parameter_number=NOTIFICATION_PARAMETER_NUMBER,
+    configuration_value=getNotificationValue(device),
+    size=4
+  })
+  local send_configuration = function()
+    dev:send(config)
+  end
+  device.thread:call_with_delay(1,send_configuration)
+end
+
+local function set_color_temperature(driver, device, command)
+  device:emit_event(capabilities.colorControl.hue(100))
+  device:emit_event(capabilities.colorTemperature.colorTemperature(command.args.temperature))
+  device:emit_event(capabilities.switch.switch("on"))
+  local dev = device:get_parent_device()
+  local config = Configuration:Set({
+    parameter_number=NOTIFICATION_PARAMETER_NUMBER,
+    configuration_value=getNotificationValue(device, 100),
+    size=4
+  })
+  local send_configuration = function()
+    dev:send(config)
+  end
+  device.thread:call_with_delay(1,send_configuration)
+end
+
+local function switch_level_set(driver, device, command)
+  if device.network_type ~= st_device.NETWORK_TYPE_CHILD then
+    local level = utils.round(command.args.level)
+    level = utils.clamp_value(level, 0, 99)
+
+    --device:emit_event(level > 0 and capabilities.switch.switch.on() or capabilities.switch.switch.off())
+
+    device:send(SwitchMultilevel:Set({ value=level, duration=command.args.rate or "default" }))
+
+    device.thread:call_with_delay(3, function(d)
+      device:send(SwitchMultilevel:Get({}))
+    end)
+  else
+    device:emit_event(capabilities.switchLevel.level(command.args.level))
+    device:emit_event(capabilities.switch.switch(command.args.level ~= 0 and "on" or "off"))
+    local dev = device:get_parent_device()
+    local config = Configuration:Set({
+      parameter_number=NOTIFICATION_PARAMETER_NUMBER,
+      configuration_value=getNotificationValue(device),
+      size=4
+    })
+    local send_configuration = function()
+      dev:send(config)
+    end
+    device.thread:call_with_delay(1,send_configuration)
+  end
+end
+
+local function can_handle_inovelli_vzw32(opts, driver, device, ...)
+  for _, fingerprint in ipairs(INOVELLI_VZW32_SN_FINGERPRINTS) do
+    if device:id_match(fingerprint.mfr, fingerprint.prod, fingerprint.model) then
+      local subdriver = require("inovelli-vzw32-sn")
+      return true, subdriver
+    end
+  end
+  return false
+end
+
+local device_init = function(self, device)
+  if device.network_type ~= st_device.NETWORK_TYPE_CHILD then
+    device:set_component_to_endpoint_fn(component_to_endpoint)
+    device:set_endpoint_to_component_fn(endpoint_to_component)
+    --initialize(device)
+  end
+end
```

Comment:
you should no longer need to override the base driver's `init` function now (or the endpoint/component mapping functions)